data/campaigns/Heir_To_The_Throne/scenarios/05a_Muff_Malal_Peninsula.cfg

```diff
@@ -1,4 +1,40 @@
 #textdomain wesnoth-httt
+
+#define ADVISORS
```

Comment:
When you have something duplicated in two or more scenarios, it's probably better to move it into one of the general macro files under _utils/_ or _macros/_ (whichever HttT calls it).